src/pages/components/cards.mdx

```diff
@@ -146,6 +146,13 @@ take the user to different pages. Static cards can appear with or without a bord
 
 </Row>
 
+#### Logo
+
+The card can feature a logo at the top of the card. The logo can be used to call out brand success stories or case studies.
+
+The logo image itself should be contained inside a space (2:1 aspect ratio). The container size is fixed at all the breakpoints. A logo grid is available in the [design specifications](https://ibm.ent.box.com/folder/95514001950?s=qqw0soczjpawvuh9j3x19fmcoqk1zsw7).
+The logo is best displayed with either a transparent background or a background color that matches the card background.
```

Comment:
@RichKummer, I'm going to leave the review of this paragraph to @oliviaflory. Thanks!

Reply:
@RichKummer I think these are all really good points, I wonder if we could add some additional details and break it up a bit so it's more digest-able to a first time reader and add some images?

The logo image is contained within a set aspect ratio of 2:1 and is a fixed size at all breakpoints. A logo grid template is available in the card design specification file to help place the logos correctly within the image container and ensure all logos appear at an equal visual hierarchy. 

[IMAGES] 1. Logo card 2. Logo with grid overlay to show what we mean by set within a 2:1 aspect ratio
![Screen Shot 2021-12-20 at 4 15 35 PM](https://user-images.githubusercontent.com/45692622/146834195-cc6997aa-890f-411d-a13d-4f158e87e06a.png)
![Screen Shot 2021-12-20 at 4 17 38 PM](https://user-images.githubusercontent.com/45692622/146834199-04d999da-aedc-46db-822c-0769768260a6.png)


The logo is best displayed with a transparent background, a background color that matches the card color can be used when necessary, see the [theme guidance] to set the correct color when producing the logo images. Follow any brand guidance when using company logos and consider what color the logo will be placed on and provide an image that will be accessible for the user. See the [Logo grid tips and techniques](https://www.ibm.com/standards/carbon/components/logo-grid#tips-and-techniques) for additional logo usage guidance.

> I noticed in our own storybook example we need to provide another image for Microsoft on the dark theme.

> I also would caution us saying the images could be set on a colored background...I think it could get weird and possibly become an issue if we ever change the token colors and get a visible difference.

[IMAGES] Do/Don't images? 1. logo on white 2. example on g90 or g100 
![Screen Shot 2021-12-20 at 3 46 56 PM](https://user-images.githubusercontent.com/45692622/146834342-c2cc93aa-954e-4e27-8ab1-3a46ef99b77f.png)
![Screen Shot 2021-12-20 at 3 47 07 PM](https://user-images.githubusercontent.com/45692622/146834352-8d2b933d-ac0b-4bc0-aa37-1fdf6eee0ca0.png)

Reply:
@RichKummer I am also curious if you think the logo template within the Sketch file is sufficient... it's a bit vague IMO, but would love your thoughts!